### PR TITLE
Add Recaptcha Verification

### PR DESCRIPTION
**Pull request by fatlotus**
Test URL: https://add-recaptcha-dot-caravel-code-reviews.appspot.com.

I've rushed this into prod, but I'd still like approval before merging into master.

### CODE REVIEW

File: caravel/__init__.py [hunk 1]
```diff
@@ -7,15 +7,20 @@
...
     from google.appengine.ext import testbed
     testbed = testbed.Testbed()
     testbed.activate()
     testbed.init_all_stubs()
+    app.testing = True
 
-from flask import Flask
-app = Flask(__name__)
+app.config["RECAPTCHA_PUBLIC_KEY"] = "6LeeKBITAAAAAPVBZTwJPsfBBIwp68QKG5tMQOz4"
+app.config["RECAPTCHA_PRIVATE_KEY"] = "6LeeKBITAAAAAEW1o535C9A7AE2dNauMcay5_2Jr"
```
Comment (georgeteo) [inline caravel/__init__.py]: Do we want the private key visible to anybody who looks at the code?

**Comment by georgeteo** [comment]
Looks good except for the privkey concern above.

**Comment by fatlotus** [comment]
Whoops! Deleted it and moved it into config, but I guess I forgot the one in init. In any case I've killed the key and generated a new one and will update the commit shortly.

**Comment by fatlotus** [comment]
Updated!

**Pull request merged**